styles.css

```diff
@@ -1238,19 +1317,16 @@ label {
 
   #contact {
     flex-direction: column;
-    width: 50%;
+    width: 40%;
     padding: 0;
     justify-content: center;
-    align-items: center;
+    align-items: start;
     position: relative;
     left: 10em;
   }
 
   #contact > h2 {
-    text-align: center;
```

Comment:
Hey bro!

Please, consider changing the "text-align" property from center to left, so the Contact Header aligns that way.

![image](https://user-images.githubusercontent.com/96673631/230812357-de335170-6402-4a59-b8f0-bda7b2f305e0.png)

Reply:
Thanks for your suggestion and I'm sorry but I think you had an error when you cloned the repository because the changes are resolved I suggest cloning again
![Captura de pantalla (192)](https://user-images.githubusercontent.com/119135868/230982055-c24f6c57-f24e-4f60-a844-21b05845307c.png)